### ISSUE TITLE
1.33 insiders untitled files missing contents

### ISSUE THREAD

**Issue by msftrncs** [Microsoft/vscode#70111]
I purposely left a untitled dirty file in a tab with my instance of Insiders, and when it updated this evening, the untitled file came up blank (thus no longer dirty).  It had updated in the last couple of days, I use it daily and check for updates each day.

Is this just a continuation of the issue when updating to 1.32?

```
Version: 1.33.0-insider (user setup)
Commit: 9b849d9de7168eb4f685823bf9b9e27a0a8dae74
Date: 2019-03-08T06:59:35.464Z
Electron: 3.1.3
Chrome: 66.0.3359.181
Node.js: 10.2.0
V8: 6.6.346.32
OS: Windows_NT x64 10.0.18351
```

**Comment by mjbvz** [comment]
Duplicate of #70006

**Comment by msftrncs** [comment]
@mjbvz, I think you might have referenced the wrong issue.    I want to reiterate that the update of my Insiders instance was AFTER the described issue on 1.32 was supposedly fixed in 1.32.1 (except for the double window which I have never experienced), but maybe the Insiders wasn't up to date with yet with 1.32.1 at that time.  My Insiders instance was up to date just a day (actually, that AM) prior to the release of 1.32, so I wouldn't have expected to see this issue reproduce there the next day.
